src/util/address.rs

```diff
@@ -922,6 +924,16 @@ mod tests {
         roundtrips(&addr);
     }
 
+    #[test]
+    fn test_p2tr() {
+        // stolen from https://github.com/bitcoin/bips/blob/master/bip-0086.mediawiki#Test_vectors
+        let key = hex_schnorr_key!("a60869f0dbcf1dc659c9cecbaf8050135ea9e8cdc487053f1dc6880949dc684c");
```

Comment:
This should be updated to take in the input internal key instead of the output key. We changed the since #691

Reply:
Rebased and removed `test_p2tr` since it was the same as `p2tr_from_untweaked`, added a couple of lines to the latter